### DISCUSSION TITLE
Add additional property for all log entries

### DISCUSSION THREAD

**Discussion by Mestika** [spatie/laravel-activitylog#1009] [Q&A]
I'm currently looking for an audit log/trail tool and came across this great project.
I am doing some research to see if it could be a fit for my company and our project and have some questions in that regard.

We have a SaaS platform where we use multi-tenant database architecture where we separate each client via a `client_id`. This client_id would need to be added to all **log entries** and preferable in its own column, which would require either to change this projects Migrationscript or to add a new additional migrationscript. I could of course add it manually for each log insert but it would be great to have it automatically added when a new entry is added. 

Besides, the audit trail is going to be in a separate database which for us make sense since it will not be attached to our current database (this is mainly for performance concerns and for data-integrity concerns, in case something happens to main DB or it needs to be restored, etc.). I see that I can change the database connection in the config file but can I also publish the migrationscript and change that before running it?

**Comment by Gummibeer** [comment]
hey,
recommended way is to add your own migration and use an observer for the activity model which fills your new column.
